src/main.py

```
@@ -0,0 +1,125 @@
+    task_id = str(uuid.uuid4())
+
+    task = AudioTask(audio_np)  # TODO for file in request.bucket tasks.append(AudioTask(file))
+
+
+    app.state.tasks[task_id] = {
+        "status": "pending",
+        "result": None,
+        "error": None,
+    }
+
+    async def run_task():
```

Ruff (0.12.2)

Missing return type annotation for private function `run_task`

Add return type annotation: `None`

(ANN202)